spec/features/cases/offender_sar/add_data_requests_spec.rb

```diff
@@ -51,6 +51,48 @@
     expect(last_row.total_value).to have_text '0'
   end
 
+  scenario 'total page count is correct', js: true do
```

Comment:
This spec doesn't need to use the JS driver, which is much slower than running the feature spec using the default - the rack driver. Turning this off saves nearly 2 seconds running this spec :)

Reply:
Removed in favour of your more concise test. :)